x/payment/types/wirepayfordata_test.go

```diff
@@ -47,7 +47,11 @@ func TestWirePayForData_ValidateBasic(t *testing.T) {
 
 	// pfd that has a different power of 2 square size
 	badSquareSizeMsg := validWirePayForData(t)
-	badSquareSizeMsg.MessageShareCommitment[0].K = 4
+	badSquareSizeMsg.MessageShareCommitment[0].K = 16
```

Comment:
[for reviewers] this test had to be updated to use a square size that is not 4 because after this line change:
https://github.com/celestiaorg/celestia-app/pull/666/files#diff-0535627976b096bde128716f4bbe48b835785e957a6466740da9fb8e6fe1f967R422

the first square size is `4` so `badSquareSizeMsg.MessageShareCommitment[0].K = 4` was a no-op. It took me a silly long time to realize why I broke an unrelated test while working on this